source/src_lcao/record_adj.cpp

```diff
@@ -243,48 +263,59 @@ void Record_adj::for_grid(const Grid_Technique &gt)
     ModuleBase::TITLE("Record_adj","for_grid");
 	ModuleBase::timer::tick("Record_adj","for_grid");
 	
-	ModuleBase::Vector3<double> tau1, tau2, dtau;
-	ModuleBase::Vector3<double> tau0, dtau1, dtau2;
-	
 	this->na_proc = 0;
-	for(int T1=0; T1<GlobalC::ucell.ntype; ++T1)
+	this->iat2ca = new int[GlobalC::ucell.nat];
+	for(int iat=0; iat<GlobalC::ucell.nat; ++iat)
 	{
-		for(int I1=0; I1<GlobalC::ucell.atoms[T1].na; ++I1)
 		{
-			const int iat = GlobalC::ucell.itia2iat(T1,I1);
 			if(gt.in_this_processor[iat])
 			{
+				iat2ca[iat] = na_proc;
```

Comment:
Record `ca` here to remove dependency in OpenMP region.
It mainly helps `cal_dk_k`

Reply:
What is the meaning of "ca"?

Reply:
A temporary loop count variable changes to a member of class, you can add some annotation for that.